### PR TITLE
Feature/ID-320/Read account information UI

### PR DESCRIPTION
**Pull request by JeferssonCL**
## Authors: 
- Jefersson Coronel Lavadenz @JeferssonCL 

### **What?**
- The mockup and the Read account information view were implemented, following the defining colors and styles. 
- Visualization of view.

### **How?**

- Implementation of the view.
- Implementation of images and xml files.
- Style implementation.
- Implementation of strings.

### **Why?**
- Creating the view will help us later develop the logic.

### **Code reviewers/Testers**
@VillalobosGarciaGaby  
@SamuelEscalera   

### **Trainer (General Reviewer):**
@mmoralesv

### CODE REVIEW

**Review by SamuelEscalera — APPROVED** [review]

**Comment by VillalobosGarciaGaby** [comment]
The UI view to me correct to merge it, update taiga.

File: app/src/main/java/com/isc/hermes/AccountInformation.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package com.isc.hermes;
+
+import androidx.appcompat.app.AppCompatActivity;
+
```
Comment (VillalobosGarciaGaby) [inline app/src/main/java/com/isc/hermes/AccountInformation.java:4]: Delete this space

File: app/src/main/java/com/isc/hermes/MainActivity.java [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
+
```
Comment (VillalobosGarciaGaby) [inline app/src/main/java/com/isc/hermes/MainActivity.java:5]: Delete this space plis

File: app/src/main/res/layout/activity_account_information.xml [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+                android:text="@string/account_title"
+                android:textColor="@color/silver"
+                android:textSize="16sp"
+                android:textStyle="bold"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
```
Comment (VillalobosGarciaGaby) [inline app/src/main/res/layout/activity_account_information.xml:53]: Delete this space

**Review by VillalobosGarciaGaby — APPROVED** [review]

File: app/src/main/res/layout/activity_account_information.xml (hunk 3 above)
Comment (JeferssonCL) [reply app/src/main/res/layout/activity_account_information.xml]: Done

File: app/src/main/java/com/isc/hermes/MainActivity.java (hunk 2 above)
Comment (JeferssonCL) [reply app/src/main/java/com/isc/hermes/MainActivity.java]: Done

File: app/src/main/java/com/isc/hermes/AccountInformation.java (hunk 1 above)
Comment (JeferssonCL) [reply app/src/main/java/com/isc/hermes/AccountInformation.java]: Done

**Comment by JeferssonCL** [comment]
Se ha mergeado la rama "feature/ID-320/ReadAccountInformationUI" el dia 14/06/23.

**Pull request merged**